spec/features/admin/budget_groups_spec.rb

```diff
@@ -129,12 +129,12 @@
       expect(page).to have_css("label.error", text: "Group name")
       expect(page).to have_content "can't be blank"
     end
-
   end
 
   context "Edit" do
 
     scenario "Show group information" do
+      budget.update_attribute :voting_style, "approval"
```

Comment:
It's probably best to keep the approval and knapsack specs separate, to make sure we have covered all differences and that we are not changing the behaviour of the knapsack style. How about copying this spec to an "approval" `context` with the corresponding checks?

Reply:
This is some stray code.
And now the budget group is no longer modified.
Its fields moved to budget.